pegasus/sites.v3/code.org/views/course_block.haml

```diff
@@ -1,4 +1,9 @@
-- url = CDO.studio_url("/s/#{id}")
+- if id.eql? "applab"
```

Comment:
I think == does the job. 

http://stackoverflow.com/questions/7156955/whats-the-difference-between-equal-eql-and